netlify/functions/save-discussion/index.ts

```diff
@@ -0,0 +1,98 @@
+import { Discussion } from './Discussion';
+import GitHubRepository from './GithubReposity';
+import { WebClient } from '@slack/web-api';
+import axios from 'axios';
+import querystring from 'querystring';
+import { HandlerEvent, HandlerContext } from '@netlify/functions';
+
+const client = new WebClient(process.env.SLACK_TOKEN);
+let githubReposity: GitHubRepository;
+let discussion: Discussion;
+
+const handler = async (event: HandlerEvent, context: HandlerContext) => {
+  if (event.httpMethod != 'POST' || !event.body) {
+    return {
+      statusCode: 400,
+    };
+  }
+  const payload = JSON.parse(
+    querystring.parse(event.body || '{}')?.payload as string
+  );
```

Comment:
Why you have to parse query-string? Do we use any queries in request?

Reply:
Ok, thanks for clarification! Maybe add comment about that, because in the first look it's very weird 😆